eel_hole/__init__.py

```diff
@@ -315,9 +354,9 @@ def search():
         log.info("search", url=request.full_path, query=query)
 
         if query:
-            resources = run_search(ix=index, raw_query=query)
+            resources = run_search(ix=search_index, raw_query=query)
         else:
-            resources = sorted_resources
+            resources = sorted(all_resources, key=__sort_resources_by_name)
```

Comment:
Will moving the sort into the search() body incur a performance penalty big enough to care about?

Reply:
This is an artifact of the previous approach where we built different indices and packages for the ferc-enabled/pudl-only options. Fixing!